tests/test_integration_queue.py

```diff
@@ -0,0 +1,200 @@
+import json
+import os
+import requests
+
+from tests.utils import json_default, create_unlock_code_request, is_valid_uuid, create_unlock_code_activation, \
+    generate_uuid, create_unlock_code_revocation, create_tax_number_validity, create_send_est
+
+ERICA_TESTING_URL = os.environ.get("ERICA_TESTING_URL", "http://localhost:8000")
+
+
+class TestV2Ping:
```

Comment:
I think we should also add a test for the get_tax_offices endpoint.

Reply:
Added!